api/views/CloudRecordSummaryView.py

```diff
@@ -86,6 +90,18 @@ def get(self, request, format=None):
          end_date,
          global_user_name) = self._parse_query_parameters(request)
 
+        # Check that at most one of group_name, service_name
+        # and global_user_name is set
```

Comment:
Why? (Answer - something along the lines of, that having more than one defined is currently ambiguous while retrieval against only one parameter is supported.)